src/styles/index.scss

```diff
@@ -1,3 +1,98 @@
-body {
+@import "stars";
+
+$color--text: #060b35;
+$color--text--title: #616070;
+$color--button: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+.product-cards {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
+  width: 200px;
```

Comment:
It would be great to create variables for all main values (main sizes, colors etc)